0000-relationship-between-protocols/0000-relationship-between-protocols.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
+
+## Motivation
+Because Interledger Protocol consists of several protocols and some related specifications, it is slightly difficult to grasp the entire relationship between the protocols. It is important for those who concretely implement ILP, to explicate the relationship so that they could quickly imagine how they should go forward. This document aims to build their mental models for ILP. A mental model is what a reader assumes something to be, reading some explanation. The mental model helps the reader predicting the behavior of what is explained even if the details are not given at that time.
+
+## Scope
+The scope of this document is the explication of the relationship between the Interledger protocols, and the details for it are out of scope. Some references are given for it though. i.e. This document draws a whole, big picture of ILP.
+
+## Perspective
+First of all, the diagram below shows the entire relationship between the protocols.
+
+![Perspective](images/perspective.svg)
```

Comment:
This diagram has a lot of good stuff in it but I find it a bit overwhelming to look at. What do you think about having smaller breakout diagrams (for example the components of a single node or the connection between just the sender and connector) and then maybe having the overall picture at the end? That would help the reader figure out what to focus on as they go through the diagram.

Also, nitpick: you mention that a node can be a sender, connector, or receiver in the beginning but only connectors are labeled as nodes in this diagram.

Reply:
What I wanted to achieve here was `the whole picture that readers can understand the entire ILP model at a single glance`. So, I would like to do carefully, the modification that you suggested. Give me time, I'll try.